app/src/main/res/values/strings.xml

```diff
@@ -539,9 +548,22 @@
     <string name="dialog_message_confirm_name_deletion">Do you want to delete %s?</string>
     <string name="edit_group">Edit Group</string>
     <string name="updating_group_please_wait">Updating Group, please wait...</string>
+    <string name="updating_teller_please_wait">Updating Teller, please wait...</string>
     <string name="please_verify_following_group_task">Please verify the following tasks before you can %1$s this group</string>
     <string name="change_group_status">Change the status of group</string>
+    <string name="change_teller_status">Change the status of Teller</string>
     <string name="please_wait_updating_group_status">"Updating group status, please wait... "</string>
     <string name="error_while_updating_group_status">Error while updating group status</string>
     <string name="error_while_creating_group">Error while creating group</string>
+    <string name="error_while_creating_teller">Error while creating Teller</string>
+    <string name="error_while_updating_teller">Error while updating Teller</string>
+    <string name="assigned_employee">Assigned Employee</string>
+    <string name="cheaques_recievable_amount">Cheaques Recievable Amount</string>
+    <string name="cash_over_short_account">Cash over Short Account</string>
+    <string name="cash_withdraw_limit">Cash Withdraw Limit</string>
+    <string name="denomination_required">Denomination Required?</string>
+    <string name="vaultaccountidentifier">VaultAccountIdentifier</string>
+    <string name="telleraccountidentifier">TellerAccountIdentifier</string>
```

Comment:
If some texts like this will not be translated then add the `translatable = false` attribute to the strings

Reply:
Okay.. I will make this change and push a commit.